boot/zephyr/Kconfig

```diff
@@ -601,6 +601,15 @@ config BOOT_ENCRYPT_X25519
 	help
 	  Hidden option selecting x25519 encryption.
 
+config BOOT_HMAC_SHA512
+	bool "Use SHA512 for HMAC/HKDF"
+	depends on BOOT_ENCRYPT_X25519
+	depends on BOOT_USE_PSA_CRYPTO
+	help
+	  By default SHA512 is used for HKDF/HMAC in key exchange expansion
```

Comment:
help: By default <b>SHA256</b> is used...

Reply:
Sorry I am stuck at PSA of nrf54

Reply:
done